frontend/src/views/Pages/UserProfile/UserCard_CoinAnimation.spec.js

```diff
@@ -45,5 +47,84 @@ describe('UserCard_CoinAnimation', () => {
     it('has an edit BFormCheckbox switch', () => {
       expect(wrapper.find('.Test-BFormCheckbox').exists()).toBeTruthy()
     })
+
+    describe('enable with success', () => {
+      beforeEach(async () => {
+        await wrapper.setData({ CoinAnimationStatus: false })
+        mockAPIcall.mockResolvedValue({
+          data: {
+            updateUserInfos: {
+              validValues: 1,
+            },
+          },
+        })
+        await wrapper.find('input').trigger('change')
+      })
+
+      it('calls the updateUserInfos mutation', () => {
+        expect(mockAPIcall).toBeCalledWith({
+          mutation: updateUserInfos,
+          variables: {
+            coinanimation: false,
+          },
+        })
+      })
+
+      it('updates the store', () => {
+        expect(storeCommitMock).toBeCalledWith('coinanimation', false)
+      })
+
+      it('toasts a success message', () => {
+        expect(toastSuccessMock).toBeCalledWith('settings.coinanimation.False')
+      })
+    })
+
+    describe('disable with success', () => {
+      beforeEach(async () => {
+        await wrapper.setData({ CoinAnimationStatus: true })
```

Comment:
Same here. I expect `false`

Reply:
Here I think you are wrong.
Line 84 sets the Data to enabled, so in line 92 a change can be triggered to disable the animation.
Line 85 mocks a successful backend call.

Reply:
You are correct. I corrected this - this was a problem with the dependency with the store in the component.
Please investigate the newsletter tests (my copy material) - I believe the error exists there aswell.

Reply:
Good catch. What to do? We merge this as it is and create a following ticket for both tests? @ulfgebhardt